speed-transition.lua

```diff
@@ -52,13 +53,21 @@ function speed_transition(_, sub)
          mark = speedup_zone_begin
          speedup_zone_end = mark+nextsub
          if shouldspeedup then
-            normalspeed = mp.get_property("speed")
-            if mp.get_property_native("video-sync") == "audio" then
-               mp.set_property("video-sync", "desync")
+            if skipmode and (nextsub-leadin>=leadin or nextsub==0) and mp.get_property("pause") == "no" then
+               if nextsub>set_timeout() or nextsub==0 then
+                  mp.command("no-osd seek "..tostring(mp.get_property("demuxer-cache-duration")-leadin).." relative exact")
```

Comment:
why `mp.get_property("demuxer-cache-duration")` instead of `set_timeout()`?

Reply:
Because set_timeout is merely the upper limit to the cache. If the cache is not full (i.e. slow network) then we may seek past a sub that is not cashed yet.